src/alexandria3k/data_sources/uspto.py

```diff
@@ -647,7 +670,7 @@ class Uspto(DataSource):
     def __init__(
         self,
         uspto_directory,
-        sample=lambda n: True,
+        sample=lambda x: (True, True),
```

Comment:
Please clarify in the line 649 comment and adjust the default value if needed, if this is a tuple of callables or a callable returning a tuple. If this is a tuple of callables, shouldn't it be `(lamda n: True, lambda n: True)`?

Reply:
A 'tuple' object is not callable, thus it is unable to accesses the callable inside the tuple.

That's why I have set a sample to be a lambda function that returns the tuple of callables.  Do you think think this should be running differently?

I will fix my description and add a comment to clarify why I access the contents of the tuple this way.

Reply:
I am asking to clarify the wording: "Is a callable that returns a tuple of callables", which I think is wrong; see below.

* Callable returning a tuple of values: `lambda x: (True, True)`
* Tuple of callables: `(lambda x: True, lambda x: True)`
* Callable that returns a tuple of callables: `lambda x: (lambda y: True, lambda y: True)`

Reply:
Yes, you are right. 

I uploaded it now in the form of **a callable that returns a tuple with two callable functions inside**.

Sorry for the misunderstanding.